pom.xml

```diff
@@ -118,8 +154,17 @@
                 <plugin>
                     <groupId>org.jenkins-ci.tools</groupId>
                     <artifactId>maven-hpi-plugin</artifactId>
+                    <extensions>true</extensions>
                     <configuration>
-                        <pluginFirstClassLoader>true</pluginFirstClassLoader>
+                        <pluginFirstClassLoader>false</pluginFirstClassLoader>
```

Comment:
Leaving this value as true, the class jenkins.plugins.openstack.compute.OpenStack wont get loaded by the CPS classLoader